ruby/vampires.rb

```diff
@@ -0,0 +1,81 @@
+puts "How many employees will be processed in this session?"
+employee_number = gets.chomp
+employee_number = employee_number.to_i
+
+while employee_number >= 1
+
+puts "What is your name?"
+name = gets.chomp!
+
+puts "What year were you born? How old are you?"
+age_answer = gets.chomp
+birth_year = age_answer.split.first
+birth_year = birth_year.to_i
+user_age = age_answer.split.last
+user_age = user_age.to_i
+calculated_age = 2017 - birth_year
+
+puts "Our company cafeteria serves garlic bread. Should we order some for you?"
+bread_answer = gets.chomp
+  if bread_answer == "yes"
+    likes_garlic = TRUE
+  elsif bread_answer == "no"
+    likes_garlic = FALSE
+  end
+
+puts "Would you like to enroll in the company’s health insurance?"
+insurance_answer = gets.chomp
+  if insurance_answer == "yes"
+  wants_insurance = TRUE
+    elsif
+    insurance_answer == "no"
+    wants_insurance = FALSE
+  end
+
+puts "Please list your allergies, one at a time and type 'done' when you're finished."
+allergy = gets.chomp
+until allergy == "sunshine" || allergy == "done"
+puts "Please list your allergies, one at a time:"
+allergy = gets.chomp
+end
+
+
+  if
+    name == "Drake Cula" || name == "Tu Fang"
+    puts "#{name} is definitely a vampire."
+  elsif
+    calculated_age == user_age && likes_garlic == TRUE && wants_insurance == TRUE
+    puts "#{name} is probably not a vampire."
+  elsif
+    calculated_age != user_age && (likes_garlic == FALSE || wants_insurance == FALSE)
+    puts "#{name} is almost certainly a vampire."
+  elsif
+    (calculated_age != user_age && likes_garlic == FALSE && wants_insurance == FALSE) || allergy == "sunshine"
```

Comment:
To follow the same pattern above, you could assert the negative: 

`(calculated_age != user_age && !likes_garlic && !wants_insurance) || allergy == "sunshine"`